tests/network-tests/src/fixtures/council/ElectCouncilFixture.ts

```diff
@@ -160,10 +172,20 @@ export class ElectCouncilFixture extends BaseQueryNodeFixture {
     await this.api.untilCouncilStage('Idle')
 
     const councilMembers = await api.query.council.councilMembers()
+    this.councilMembersIds = councilMembers.map((m) => m.membershipId)
     assert.sameMembers(
-      councilMembers.map((m) => m.membershipId.toString()),
+      this.councilMembersIds.map((id) => id.toString()),
       candidatesToWinIds
     )
+
+    // change accounts to known accounts
+    const oldCouncilMemberAccounts = await this.getCouncilMembersControllerAccounts()
+    const newCouncilMemberAccounts = await this.api.updateCouncillorsAccounts(
+      oldCouncilMemberAccounts,
+      this.councilMembersIds
+    )
```

Comment:
`updateCouncillorsAccounts` can query `api.query.council.councilMembers`, so that they don't need to be passed as an argument.